lib/install/examples/stimulus/hello_stimulus.js

```diff
@@ -0,0 +1,17 @@
+// Run this example by adding <%= javascript_pack_tag 'hello_stimulus' %> 
+// to app/views/layouts/application.html.erb.
+// It will render a <div>Hello Stimulus</div> at the bottom of the page.
+
+import { Application } from "stimulus"
+import { definitionsFromContext } from "stimulus/webpack-helpers"
+
+const application = Application.start()
+const context = require.context("./controllers", true, /\.js$/)
+application.load(definitionsFromContext(context))
+
+document.addEventListener('DOMContentLoaded', () => {
+  var hello = document.createElement('div')
+  hello.setAttribute("data-controller", "hello")
+  hello.setAttribute("data-target", "hello.output")
+  document.body.appendChild(hello)
+})
```

Comment:
Creating an element with JavaScript isn't a very good introduction to Stimulus since you'll almost always be working with server-rendered HTML. Let's ditch this file and start people out right by appending the setup code to `app/javascript/packs/application.js`:
```js
import { Application } from "stimulus"
import { definitionsFromContext } from "stimulus/webpack-helpers"

const application = Application.start()
const context = require.context("controllers", true, /\.js$/)
application.load(definitionsFromContext(context))
```

Reply:
👌 ok, ready for another look. 

Appended the setup code to `app/javascript/packs/application.js` and ditched the previous file.